src/main/twirl/uk/gov/hmrc/govukfrontend/views/components/GovukHeader.scala.html

```diff
@@ -54,7 +54,7 @@
           </svg>
           <!--<![endif]-->
           <!--[if IE 8]>
-          <img src="@calculateAssetPath(assetsPath, "images/govuk-logotype-crown.png")" class="govuk-header__logotype-crown-fallback-image" width="36" height="32">
+          <img src="@routes.Assets.at("images/govuk-logotype-crown.png")" class="govuk-header__logotype-crown-fallback-image" width="36" height="32">
```

Comment:
Ah, I think I had intended that instead of using the images in the resources directory we would remove those images and use the ones in the hmrcfrontend webjar instead (via the reverse router)

Reply:
@matthewmascord Worry not, this is still a WIP! I am making that change now to delete the images in the `/assets` folder, and use the ones from `govuk-frontend` instead.